keyboards/planck/keymaps/prog_qgmlwb/scripts/config.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Honestly, I'm not really a fan of having these scripts present, as a lot of this is outrigh handled by `qmk compile` and `qmk flash`, and the internal make processes.